<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>476</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Outlier define via LOF</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/header.png</normaloff>res/header.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="plainText">
       <string>Drag and drop LAS/LAZ file or SQLITE DB with .db or .sqlite extension here; set number of neighbours (KNN) and LOF threshold. 
For SQLITE also select colums to use, if one column is not set it will be assigned a value of &quot;0.0&quot; - in case you want to process 2D data.   </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="horizontalGroupBox">
      <property name="title">
       <string>LOF parameters</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="kspin">
         <property name="prefix">
          <string>KNN:   </string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="tspin">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="prefix">
          <string>Threshold:   </string>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="sqlitepanel">
      <property name="title">
       <string>sqlite columns</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="xcomboBox">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ycomboBox_2"/>
       </item>
       <item>
        <widget class="QComboBox" name="zcomboBox_3"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="threshold">
      <property name="value">
       <number>15</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="buttremove">
        <property name="toolTip">
         <string>New point cloud without detected outliers</string>
        </property>
        <property name="text">
         <string extracomment="New point cloud without detected outliers">Remove outliers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttkeep">
        <property name="toolTip">
         <string>New point cloud with detected outliers</string>
        </property>
        <property name="text">
         <string extracomment="New point cloud with detected outliers">Keep only outliers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttassign">
        <property name="toolTip">
         <string>Keep point cloud with scalar with LOF value</string>
        </property>
        <property name="text">
         <string extracomment="Keep point cloud with scalar with LOF value">Assign LOF</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
